ttnn/cpp/ttnn/operations/matmul/device/matmul_op.cpp

```diff
@@ -920,9 +973,12 @@ Matmul create_matmul_struct(
     auto arch = input_tensor_a.device()->arch();
     const bool has_user_grid = parameters.user_core_coord.has_value();
     const bool has_program_config = parameters.program_config.has_value();
-    const auto increase_fidelity = !has_program_config && !has_user_grid;
+    bool are_inputs_low_precision_df =
```

Comment:
is this unused parameter?

Reply:
it's used to determine the fidelity

Reply:
oh, completely missed `increase_fidelity`, thanks for pointing out.